opus/application/static_media/js/browse.js

```diff
@@ -559,23 +536,69 @@ var o_browse = {
                         break;
                     case 39:  // next
                         detailOpusId = $("#galleryView").find(".op-next").data("id");
+                        obsNum++;
                         o_browse.removeEditMetadataDetails();
                         o_browse.loadPageIfNeeded("next", detailOpusId);
                         break;
                     case 37:  // prev
                         detailOpusId = $("#galleryView").find(".op-prev").data("id");
+                        obsNum--;
                         o_browse.removeEditMetadataDetails();
                         o_browse.loadPageIfNeeded("prev", detailOpusId);
                         break;
+                    case 38:  // up
+                        // note that if there is no where to go in the up direction, nothing will change ... it could go to the very first, tho
```

Comment:
I don't understand this comment. What does "it could go to the very first" mean? Also "nowhere" is one word.